app/_locales/zh/messages.json

```diff
@@ -1277,6 +1295,9 @@
   "functionApprove": {
     "message": "功能：批准"
   },
+  "functionSetApprovalForAll": {
+    "message": "功能：SetApprovalForAll"
```

Comment:
I am wondering why `SetApprovalForAll` is preserved for some and not other `functionSetApprovalForAll` translations, and why `Approve` in `functionApprove` is translated